client/src/pages/selectPlayers.tsx

```diff
@@ -46,7 +38,26 @@ const useStyles = makeStyles(() => ({
   },
 }));
 
-export default function ChoosePlayers() {
+export default function SelectPlayers() {
```

Comment:
Need help with naming of this page... thoughts?

Reply:
maybe something like Dashboard or Main page?

Reply:
I went with "Selection" for now; but we can definitely always change it. I am wondering if we could use "Dashboard" for that initial page where users can choose which branch.